Changes

```diff
@@ -2,6 +2,9 @@ Revision history for Perl module Mojo::WebSocketProxy
 
 {{$NEXT}}
 
+          2019-09-20 01:44:27+00:00 UTC
```

Comment:
Needs removing this timestamp (it should be supplied by the `{{$NEXT}}` line above automatically upon release.)

Reply:
ok,